feature-libs/customer-ticketing/styles/components/_customer-ticketing-list.scss

```diff
@@ -0,0 +1,76 @@
+cx-customer-ticketing-list {
+  .cx-table {
+    @include media-breakpoint-down(lg) {
+      display: none;
+    }
+
+    thead {
+      border-top: 1px solid var(--cx-color-light);
+
+      th {
+        text-transform: none;
+        font-weight: 600;
+        font-size: 14px;
+        line-height: 20px;
+        color: #000000;
+        text-align: left;
+        padding: 16px 0;
+      }
+    }
+    tbody {
+      tr {
+        td {
+          font-size: 14px;
+          line-height: 20px;
+          color: #000000;
+          padding: 16px 0;
+        }
+      }
+    }
+  }
+
+  .cx-ticketing-list-mobile {
```

Comment:
Do we really need these?  I will be surprised if we do not have mobile breakpoints already defined.

Reply:
this type of breakpoints are the ones used across the code in different places, if there is a better implementation we can change it too!